relayer-cli/src/commands/query/channels.rs

```diff
@@ -258,6 +301,20 @@ impl QueryChannelsOutput {
         Self::Summary(Vec::new())
     }
 
+    fn pretty() -> Self {
+        Self::Pretty(Vec::new())
+    }
+
+    fn push_pretty(&mut self, pe: PrettyOutput) {
+        assert!(matches!(self, Self::Pretty(_)));
```

Comment:
This assertion probably won't throw a helpful error if the assertion fails. Was there a rationale for checking this using the `assert!` macro? The `if let` is also performing the same check, but instead of the `else` branch calling `unreachable!`, perhaps we could output a more useful error instead?

Reply:
Indeed, this `assert!` will not be helpful. I just followed the format of `push_summary` and `push_verbose` to respect the current structure. 
The latest commit modified that.